state/state.go

```diff
@@ -586,14 +586,18 @@ func (state *HelmState) UpdateDeps(helm helmexec.Interface) []error {
 // 	 be constructed relative to the `base path`.
 // - Everything else is assumed to be an absolute path or an actual <repository>/<chart> reference.
 func normalizeChart(basePath, chart string) string {
-	regex, _ := regexp.Compile("^[.]?./")
-	if !regex.MatchString(chart) {
+	if !isLocalChart(chart) {
 		return chart
 	}
 	return filepath.Join(basePath, chart)
 }
 
 func isLocalChart(chart string) bool {
+	regex, _ := regexp.Compile("^[.]?./")
```

Comment:
@mumoshu Is there a reason a `/` is required to be a local chart?  Its seems valid to have the helmfile.yaml in the same folder as a local chart:

```
├── Chart.yaml
├── charts
│   ├── grafana-1.14.7.tgz
│   ├── influxdb-0.11.0.tgz
│   ├── kapacitor-1.1.0.tgz
│   ├── telegraf-ds-1.7.4.tgz
│   └── telegraf-s-1.7.4.tgz
├── helmfile.d
│   └── helmfile.yaml
```
And the helmfile.yaml:
```
releases:
- name: myrelease
  chart: ..
```
I'd expect the regex to just be `^[.]`.

Reply:
For now, I can work around this by adding a trailing slash `/` to my `chart` path.